shared/fs/common/index.stories.js

```diff
@@ -20,18 +21,25 @@ const pathItemActionPopupProps = (path: Types.Path) => {
     name: Types.getPathNameFromElems(pathElements),
     path,
     pathElements,
-    menuItems: [
-      {
-        title: 'menu item',
-        onClick: Sb.action('onClick'),
-      },
-    ],
+    ...(isMobile
+      ? {
+          saveMedia: Sb.action('saveMedia'),
+          shareNative: Sb.action('shareNative'),
+        }
+      : {}),
```

Comment:
what happened to `menuItems`?

Reply:
They were gone long ago; we now construct them in the component. Container still decides whether an item shows up in the menu, but by either populating an action or not, instead of constructing menu items directly.